src/python/pants/option/options.py

```diff
@@ -317,14 +317,25 @@ def for_scope(self, scope, inherit_from_enclosing_scope=True):
 
     return values
 
-  def get_fingerprintable_for_scope(self, scope):
+  def get_fingerprintable_for_scope(self, scope, include_passthru=False):
```

Comment:
What I mean is that `self.passthru_args_for_scope()` returns an empty list if the task doesn't support passthru args, in which case `pairs.extend(...)` below will be a no-op. So do we need to add the `include_passthru` parameter to this method at all? I'm not sure what behavior change dropping it would cause.